core/src/main/java/org/infinispan/xsite/irac/IracManagerKeyState.java

```diff
@@ -15,7 +15,24 @@
  *
  * @since 14
  */
-interface IracManagerKeyState extends IracManagerKeyInfo {
+interface IracManagerKeyState {
+
+   IracManagerKeyInfo getKeyInfo();
+
+   /**
+    * @return The key.
+    */
+   Object getKey();
```

Comment:
@pruivo I kept these methods for convenience so that we didn't have to access the values via `getKeyInfo` all of the time. I'm happy to remove if you prefer though.

Reply:
that's fine. :+1: